raft-tech/TANF-app
# Bug/1772 errors when submitting files

## Summary of Changes
_Provide a brief summary of changes_
Pull request closes #1772 

## How to Test
_List the steps to test the PR_
These steps are generic, please adjust as necessary.

1. Changes must be tested on deployed environments -- this functionality worked locally.
1. After logging in to TDP, proceed to upload a file and verify that you receive green success message.
2. Verify via Django Admin that data files are still scanned from CLAMAV

> *Demo GIF(s) and screenshots for testing procedure*
![image](https://user-images.githubusercontent.com/84722778/168163399-413d300d-48ec-4626-a24d-9698ffbf18d4.png)

## Tabletop


## Deliverables
_More details on how deliverables herein are assessed included [here](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:
+ [x] Users are able to upload files to deployed sites
+ [x] Users can still download files
+ [x] Files are still scanned from CLAMAV

### [Deliverable 2: Tested Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
+ Are code coverage minimums met?

### [Deliverable 3: Properly Styled Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)


### [Deliverable 4: Accessible](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)



### [Deliverable 5: Deployed](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)


### [Deliverable 6: Documented](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)


### [Deliverable 7: Secure](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)


### [Deliverable 8: User Research](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
